cluster/grpc_rpc_client.go

```diff
@@ -272,10 +272,10 @@ func (gs *GRPCClient) getServerHost(sv *Server) (host, portKey string) {
 	}
 
 	if gs.infoRetriever.Region() == serverRegion {
-		logger.Log.Warnf("server %s is in same region, using internal host", sv.ID)
+		logger.Log.Infof("server %s is in same region, using internal host", sv.ID)
```

Comment:
Don't you think that debug level would be better?

Reply:
Imagine if all connections are failing and we are struggling to debug (in a moment we can't put log level to debug). This will help a lot. Besides, it is only logged when a new server is up and all the others connect to it.

Reply:
Ok